From 6f9e5c8e3d7e3251af3e047e01ae38cc26b41a4e Mon Sep 17 00:00:00 2001
From: Max Reitz <mreitz@redhat.com>
Date: Tue, 7 Jan 2014 21:57:16 +0100
Subject: [PATCH 11/14] qcow2: Add missing space in error message

RH-Author: Max Reitz <mreitz@redhat.com>
Message-id: <1389131839-12920-12-git-send-email-mreitz@redhat.com>
Patchwork-id: 56547
O-Subject: [RHEL-7.0 qemu-kvm PATCH v2 11/14] qcow2: Add missing space in error message
Bugzilla: 1033490
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

BZ: 1033490

The error message in qcow2_downgrade about an unsupported refcount
order is missing a space. This patch adds it.

Signed-off-by: Max Reitz <mreitz@redhat.com>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit 9e3f08923a14ba0655c6797edd9ffef44bb8cbf2)

Signed-off-by: Max Reitz <mreitz@redhat.com>
---
 block/qcow2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 block/qcow2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/block/qcow2.c b/block/qcow2.c
index 5b28259..3f7fd51 100644
--- a/block/qcow2.c
+++ b/block/qcow2.c
@@ -2013,7 +2013,7 @@ static int qcow2_downgrade(BlockDriverState *bs, int target_version)
          * support anything different than 4 anyway, there is no point in doing
          * so right now; however, we should error out (if qemu supports this in
          * the future and this code has not been adapted) */
-        error_report("qcow2_downgrade: Image refcount orders other than 4 are"
+        error_report("qcow2_downgrade: Image refcount orders other than 4 are "
                      "currently not supported.");
         return -ENOTSUP;
     }
-- 
1.7.1

